packages/contracts-bedrock/test/kontrol/proofs/interfaces/KontrolInterfaces.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Types } from "src/libraries/Types.sol";
+
+interface OptimismPortalInterface {
```

Comment:
Is redeclaring these interfaces necessary, i.e. can we just import and use the contracts themselves as the interface and delete this folder?

Reply:
The point of these is to isolate the `proofs` dir from the rest of the project so that we have strictly what we need for compiling and building the K definitions, avoiding any possible overhead. I'll double-check if having the contracts themselves as interfaces will include them in the K definition of the project, but I would say it does

Reply:
I would guess you're right. We plan to create interfaces for the contracts next year, so if you are right we can keep this as-is and at that time we can remove this.

I'd also just suggest changing the interface naming scheme to e.g. `IOptimismPortal` since that's what we do elsewhere